src/screenshot_gui.cpp

```diff
@@ -50,33 +48,44 @@ struct ScreenshotWindow : Window {
 	/**
 	 * Make a screenshot.
 	 * Ask for confirmation if the screenshot will be huge.
-	 * @param t Screenshot type: World, defaultzoom, heightmap or viewport screenshot
+	 * @param st Screenshot type: World, defaultzoom, heightmap or viewport screenshot
 	 */
 	static void TakeScreenshot(ScreenshotType st) {
-		ViewPort vp;
-		SetupScreenshotViewport(st, &vp);
-		if ((uint64)vp.width * (uint64)vp.height > 8192 * 8192) {
-			/* Ask for confirmation */
-			_screenshot_type = st;
-			SetDParam(0, vp.width);
-			SetDParam(1, vp.height);
-			ShowQuery(STR_WARNING_SCREENSHOT_SIZE_CAPTION, STR_WARNING_SCREENSHOT_SIZE_MESSAGE, nullptr, ScreenshotConfirmationCallback);
-		}
-		else {
-			/* Less than 64M pixels, just do it */
-			MakeScreenshot(st, nullptr);
-		}
+		::TakeScreenshot(st);
 	}
+};
 
-	/**
-	 * Callback on the confirmation window for huge screenshots.
-	 * @param w Window with viewport
-	 * @param confirmed true on confirmation
-	 */
-	static void ScreenshotConfirmationCallback(Window *w, bool confirmed) {
-		if (confirmed) MakeScreenshot(_screenshot_type, nullptr);
+static ScreenshotType _screenshot_type;
+
+/**
+ * Callback on the confirmation window for huge screenshots.
+ * @param w Window with viewport
+ * @param confirmed true on confirmation
+ */
+static void ScreenshotConfirmationCallback(Window *w, bool confirmed) {
```

Comment:
Opening { should be on a new line
(Also some other places)

Reply:
Yes, I may as well clean up these files to adhere to the coding style while I've got them open. :smile: